.github/workflows/storage.conf

```diff
@@ -0,0 +1,236 @@
+# This file is is the configuration file for all tools
```

Comment:
Maybe move this file to `misc/storage.conf`, and update `README.md` to use this configuration. (it's not a blocker for this PR).